qurator/dinglehopper/ocr_files.py

```diff
@@ -104,6 +91,35 @@ def page_extract(tree, *, textequiv_level="region"):
     return ExtractedText(None, regions, "\n", None)
 
 
+def extract_texts_from_reading_order_group(group, tree, nsmap, textequiv_level):
+    """Recursive function to extract the texts from TextRegions in ReadingOrder."""
+    regions = []
+    ro_children = group.findall("./page:RegionRefIndexed", namespaces=nsmap)
+    ro_children.extend(group.findall("./page:OrderedGroupIndexed", namespaces=nsmap))
+    ro_children = filter(lambda child: "index" in child.attrib.keys(), ro_children)
+    for ro_child in sorted(ro_children, key=lambda child: int(child.attrib["index"])):
```

Comment:
Maybe using the [get method of dict with a default value](https://docs.python.org/3.9/library/stdtypes.html#dict.get) instead of filtering children without index is more user friendly?